types/zookeeper/index.d.ts

```diff
@@ -0,0 +1,108 @@
+// Type definitions for zookeeper 3.4
+// Project: https://github.com/yfinkelstein/node-zookeeper#readme
+// Definitions by: xialeistudio <https://github.com/xialeistudio>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+/// <reference types="node" />
+
+type ACL = number | { perms: number, scheme: string, auth: string };
+
+interface Stat {
+    czxid: number;
+    mzxid: number;
+    ctime: number;
+    mtime: number;
+    version: number;
+    cversion: number;
+    aversion: number;
+    ephemeralOwner: string;
+    dataLength: number;
+    numChildren: number;
+    pzxid: number;
+}
+
+interface ConnectionOptions {
+    connect?: string;
+    timeout?: number;
+    host_order_deterministic?: boolean;
+    data_as_buffer?: boolean;
+    debug_level?: number;
+}
+
+type path_callback = (rc?: number, error?: string, path?: string) => void;
+type stat_callback = (rc?: number, error?: string, stat?: Stat) => void;
+type data_callback = (rc?: number, error?: string, data?: Buffer | string) => void;
+type child_callback = (rc?: number, error?: string, children?: string[]) => void;
+type child2_callback = (rc?: number, error?: string, children?: string[], stat?: Stat) => void;
+type void_callback = (rc?: number, error?: string) => void;
+type watch_callback = (type?: number, state?: number, path?: string) => void;
+type acl_callback = (rc?: number, error?: string, acl?: ACL[], stat?: Stat) => void;
+
+export = Zookeeper;
+
+declare class Zookeeper extends Client {
+    constructor(options: ConnectionOptions);
+
+    connect(callback: (e: Error | null, client: Client) => void): void;
+
+    close(): void;
+}
+
+declare class Client {
+    client_id?: number;
+    client_password?: any;
+    state?: any;
```

Comment:
I think the type is `number` https://github.com/yfinkelstein/node-zookeeper#output-parameters